webclient/app/src/app/commons/entityDialog/EntityDialog.jsx

```diff
@@ -240,7 +257,7 @@ function EntityDialog(props) {
         if (!entity.fields || entity.fields.length <= 0) {
             return (
                 <div className={entityEditorStyles.statementWrapper}>
-                    <Statement icon={<CheckBoxOutlineBlank/>} message={t("entityDialog.noFields")}/>
+                    <Statement message={t("entityDialog.noFields")}/>
```

Comment:
An icon is necessary here. Maybe a Plus icon or something else would be great here. https://mui.com/components/material-icons/
```suggestion
                    <Statement icon={<Add/>} message={t("entityDialog.noFields")}/>
```

Reply:
there is a default icon saved in the component